### DISCUSSION TITLE
iCloud remove

### DISCUSSION THREAD

**Discussion by worco8470** [community/community#141901] [Code Security]
### Select Topic Area

Question

### Body

We looking  Security Researcher or Bug Bounty Hunter to join our team for an exciting project.

**Comment by ghostinhershell** [comment]
Hi @worco8470, we're happy you're here 😀

However, we currently do not allow job listings on the Community Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities! In the meantime I've removed your contact information from this post and locked this post.

Thanks for your interest 👍
